client/my-sites/site-settings/manage-connection/index.jsx

```diff
@@ -59,14 +40,8 @@ class ManageConnection extends Component {
 	}
 }
 
-export default connect(
-	( state ) => {
-		const siteId = getSelectedSiteId( state );
-
-		return {
-			siteIsAtomic: isSiteAutomatedTransfer( state, siteId ),
-			siteIsJetpack: isJetpackSite( state, siteId ),
-			siteSlug: getSelectedSiteSlug( state ),
-		};
-	}
-)( localize( ManageConnection ) );
+export default flowRight(
+	( ) => {},
+	localize,
+	redirectNonJetpackToGeneral( ManageConnection )
```

Comment:
`flowRight` will call each of the first call arguments on the second argument. So you only need to pass `redirectNonJetpackToGeneral` here - you don't need to also pass `ManageConnection` to it, because `flowRight` will do that for you.